imagetest/test_suites/network/gvnic_test.go

```diff
@@ -45,8 +50,27 @@ func CheckGVNICPresentWindows(interfaceName string) error {
 	return errors.New("GVNIC not present")
 }
 
+func CheckGVNICPerformance() (string, error) {
+	// Wait for iperf to finish.
+	for i := 0; i <= maxRetries; i++ {
+		results, err := utils.GetMetadataGuestAttribute("testing/results")
```

Comment:
The test wrapper uploads its results to GCS (https://github.com/GoogleCloudPlatform/guest-test-infra/blob/master/imagetest/cmd/wrapper/main.go#L104) and thus we already have a good place to stick test data that will stick around. Instead of using guest attributes, we could either directly upload the perf results to the same location or merge them back into the junit output that gets uploaded there at the end.

Reply:
The results are being merged with the junit output with this method, since we're outputting the results of the test as part of the test. The iperf results are viewable from the results uploaded to GCS.